op-chain-ops/genesis/layer_two.go

```diff
@@ -32,7 +32,10 @@ func BuildL2Genesis(config *DeployConfig, l1StartBlock *types.Block) (*core.Gene
 		FundDevAccounts(db)
 	}
 
-	SetPrecompileBalances(db)
```

Comment:
I cannot find any usage of `op-chain-ops/genesis/testdata/test-deploy-config-devnet-l1-set-precompiled.json`. We can remove `config.setPrecompileBalances` if we don't use it. What do you think about that?

Reply:
I added the option of setPrecompileBalances in the past to be able to select to use or not! So I added the option in the devnetL1-template! (I don't know why it is removed).
So, I think we can use option!

Reply:
Ok. It doesn't matter. Originally it used 2 tests in `layer_two.go`. We don't use it now, but we might leave it in config and use it later. Thank you very much!